spec/default_module_facts.yml

```diff
@@ -0,0 +1,6 @@
+id: root
+path: 
+ipaddress_eth0: 10.0.0.1
+osfamily: Debian
```

Comment:
I think these should already be in (or should be added in?) the os specific stuff in the spec tests instead?

And we should switch to rspec-puppet-facts (but as a separate thing)